### DISCUSSION TITLE
Magpie with touchscreen

### DISCUSSION THREAD

**Discussion by Nico877** [Blinue/Magpie#426] [General]
Hello, I mainly played visual novels with Magpie, and since I have a Surface Pro 8 I really like using the touchscreen instead of mouse/keyboard. The problem is that any time I touch an area of the screen outside the windowed game, it makes the game go back to window mode and I have to start the scaler again with Magpie. Is there any way to use the touchscreen without exiting the upscaling?

**Comment by Blinue** [comment]
https://github.com/Blinue/Magpie/issues/218#issuecomment-980506414 This is an old version that supports touch. Please follow the steps to install it.

**Reply by Nico877** [reply]
Thank you, it works great. I hope you can bring back this feature in future versions :)
